pkg/logs/input/docker/launcher.go

```diff
@@ -197,6 +226,9 @@ func (l *Launcher) startTailer(container *Container, source *config.LogSource) {
 // stopTailer stops the tailer matching the containerID.
 func (l *Launcher) stopTailer(containerID string) {
 	if tailer, isTailed := l.tailers[containerID]; isTailed {
+		// One of those two RemoveInput is a no-op
+		tailer.source.RemoveInput(containerID)
+		l.collectAllSource.RemoveInput(containerID)
```

Comment:
how can we still add inputs to `collectAllSource ` ?

Reply:
Seen with you, we have to to display it properly in the status (that can be a noop) if we're removing a container that comes from AD